### ISSUE TITLE
"Invalid Price Type" error while placing Intraday SL-M order.

### ISSUE THREAD

**Issue by amit-kumar27742** [krishnavelu/alice_blue#36]
Placing SL-M order using below sample code is getting placed without error, but rejected on Alice blue web terminal with Rejected Reason "**Invalid Price Type**". Tried with other symbols and more appropriate trigger_price, but received rejection with same error. Not able to make out what does the error mean. Tried using different possible price like 0 , 0.0 , None , but in vain. Please have a look.  Note: When same order is directly placed from https://ant.aliceblueonline.com/ its going through fine.

Code below to replicate:

# TransactionType.Buy, OrderType.StopLossMarket, ProductType.Intraday

print ("%%%%%%%%%%%%%%%%%%%%%%%%%%%%8%%%%%%%%%%%%%%%%%%%%%%%%%%%%%")
print(
   alice.place_order(transaction_type = TransactionType.Buy,
                     instrument = alice.get_instrument_by_symbol('NSE', 'YESBANK'),
                     quantity = 1,
                     order_type = OrderType.StopLossMarket,
                     product_type = ProductType.Intraday,
                     price = 0.0,
                     trigger_price = 47.75,
                     stop_loss = None,
                     square_off = None,
                     trailing_sl = None,
                     is_amo = False)
)

**Comment by amit-kumar27742** [comment]
Issue Identified. OrderType.StopLossMarket.value is 'SLM', whereas it should be 'SL-M' . Unfortunately aliceblue documentation mentions it to be 'SLM' but is incorrect. You may test and change it in class definations. Thanks.

**Comment by krishnavelu** [comment]
@amit-kumar27742 thank you for the identification. I’ll test once and make the change.

**Comment by krishnavelu** [comment]
I get the perfect result without modifying it to `'SL-M'`. Test it again with only above mentioned code without any additions code of yours.

```
%%%%%%%%%%%%%%%%%%%%%%%%%%%%8%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
{'status': 'success', 'message': 'Order placed successfully', 'data': {'oms_order_id': '191212000251043'}}
```

**Comment by krishnavelu** [comment]
Closing this as of now. Reopen if you are still facing this issue.

**Comment by krishnavelu** [comment]
Fixed in #43
